pkgs/development/compilers/go/1.17.nix

```diff
@@ -229,7 +229,12 @@ stdenv.mkDerivation rec {
   '';
 
```

Comment:
1.17 is EOL and will be removed soon, please remove it from this PR.

Reply:
Done